File: v2/csharp/src/main/csharp/io/swagger/Model/Events.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace io.swagger.Model {
  public class Events {
    

    
    public List<Event> Events { get; set; }

    

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class Events {\n");
      
      sb.Append("  Events: ").Append(Events).Append("\n");
      
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  
  
}

File: v2/android/src/main/java/io/swagger/client/model/Events.java
package io.swagger.client.model;

import io.swagger.client.model.Event;
import java.util.*;

import com.wordnik.swagger.annotations.*;
import com.google.gson.annotations.SerializedName;


@ApiModel(description = "")
public class Events  {
  
  @SerializedName("events")
  private List<Event> events = new ArrayList<Event>() ;

  
  /**
   **/
  @ApiModelProperty(value = "")
  public List<Event> getEvents() {
    return events;
  }
  public void setEvents(List<Event> events) {
    this.events = events;
  }

  

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class Events {\n");
    
    sb.append("  events: ").append(events).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}
